packages/pigeon/platform_tests/shared_test_plugin_code/lib/integration_tests.dart

```diff
@@ -979,6 +988,24 @@ void runPigeonIntegrationTests(TargetGenerator targetGenerator) {
       expect(api.callFlutterNoop(), completes);
     });
 
+    // testWidgets('errors are returned from non void methods correctly',
```

Comment:
We should land them not commented out, with `skip: true` and a link to an issue explaining why.